group/views.py

```diff
@@ -1,7 +1,35 @@
 from django.shortcuts import render
 from django.http import HttpResponse
+from formtools.wizard.views import SessionWizardView
+from .forms import *
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin
 
 # Create your views here.
 
 def group(request):
-    return HttpResponse("Hello, group!")
\ No newline at end of file
+    return HttpResponse("Hello, group!")
+
+class FormWizardView(LoginRequiredMixin, SessionWizardView):
```

Comment:
can we change the name of the class to CreateGroup or CreateGroupFormWizard or something similar